tests/test_helpers.py

```diff
@@ -474,62 +474,44 @@
      ),
-    (["helper", "u_milestone", "sb_firstprojectum"],
-     "Please choose from one of the following to update/add:\n"
-     "1. new milestone\n"
-     "2. Kick off meeting    due date: 2020-05-06    status: finished\n"
-     "3. Project lead presentation    due date: 2020-05-20    status: proposed\n"
-     "4. deliverable    due date: 2020-06-01    status: converged\n"
-     "5. planning meeting    due date: 2020-06-01    status: converged\n"
-     ),
-    (["helper", "u_milestone", "sb_firstprojectum", "--verbose"],
-     "Please choose from one of the following to update/add:\n"
-     "1. new milestone\n"
-     "2. Kick off meeting    due date: 2020-05-06    status: finished\n"
-     "     audience: ['lead', 'pi', 'group_members']\n"
-     "     objective: introduce project to the lead\n"
-     "     type: meeting\n"
-     "     notes:\n"
-     "       - kickoff note\n"
-     "3. Project lead presentation    due date: 2020-05-20    status: proposed\n"
-     "     audience: ['lead', 'pi', 'group_members']\n"
-     "     objective: lead presents background reading and initial project plan\n"
-     "     type: meeting\n"
-     "     notes:\n"
-     "       - do background reading\n"
-     "       - understand math\n"
-     "4. deliverable    due date: 2020-06-01    status: converged\n"
-     "     audience: ['beginning grad in chemistry']\n"
-     "     type: meeting\n"
-     "     notes:\n"
-     "       - deliverable note\n"
-     "       - do this\n"
-     "       - do that\n"
-     "5. planning meeting    due date: 2020-06-01    status: converged\n"
-     "     audience: ['lead', 'pi', 'group_members']\n"
-     "     objective: develop a detailed plan with dates\n"
-     "     type: meeting\n"
-     "     notes:\n"
-     "       - do this\n"
-     "       - do that\n"
-     ),
-    (["helper", "u_milestone", "sb_firstprojectum", "--current"],
-     "Please choose from one of the following to update/add:\n"
-     "1. new milestone\n"
-     "3. Project lead presentation    due date: 2020-05-20    status: proposed\n"
-     "4. deliverable    due date: 2020-06-01    status: converged\n"
-     "5. planning meeting    due date: 2020-06-01    status: converged\n"
-     ),
+    (["helper", "u_milestone", "--uuid_fragment", "sb_fir", "--projectum_id", "sb_firstprojectum",
+      "--due-date", "2023-01-01", "--notes", "do this", "do that", "--type", "mergedpr"],
+     "A uuid fragment and projectum id have been entered.\n"
+     "You may enter a uuid fragment or a projectum id but not both.\n"
+     "Please enter the uuid fragment of the milestone you would like to update.\n"
+     "If you would like to add a new milestone to a projectum, enter the projectum id.\n"
+     ),
+     (["helper", "u_milestone", "--due-date", "2023-01-01",
+       "--notes", "do this", "do that", "--type", "mergedpr"],
+      "No uuid fragment or projectum id has been entered.\n"
+      "Please enter the uuid fragment of the milestone you would like to update.\n"
```

Comment:
please see the comment above and make this help message something comparable so it is clear what the user should do.